packages/subscription-manager/src/components/fields/EmailFormatInput/EmailFormatInput.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-imports */
+import { useTranslate } from 'i18n-calypso';
+import { FormEventHandler } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSelect from 'calypso/components/forms/form-select';
+import './styles.scss';
+
+export type EmailFormatType = 'html' | 'text';
+
+type EmailFormatInputProps = {
+	value: EmailFormatType;
+	onChange: FormEventHandler< HTMLSelectElement >;
+};
+
+const EmailFormatInput = ( { value, onChange }: EmailFormatInputProps ) => {
+	const translate = useTranslate();
+
+	return (
+		<FormFieldset className="email-format-input">
+			<FormLabel htmlFor="subscription_delivery_mail_option">
```

Comment:
`htmlFor="subscription_delivery_mail_option"` points to a nonexistent ID. I see it is a copy-paste mistake, I will fix it with a follow-up PR.

Reply:
True, I missed this one, I can fix it on a new PR.

Reply:
lol 😂 Do not worry guys. All the points I made are really minor, none of them are breaking. Stuff like this happens all the time. It's easy to slip something in, even when you have multiple people looking at one PR, especially with all the workload.